controller.py

```diff
@@ -805,24 +913,23 @@ class URNLookupController(CoreURNLookupController):
 
     log = logging.getLogger("URN lookup controller")
 
-    def __init__(self, _db, coverage_provider_kwargs=None):
+    def __init__(
+            self, _db, identifier_resolver_class=IdentifierResolutionCoverageProvider,
```

Comment:
I didn't end up swapping out identifier_resolver_class, but I think it'd be useful in the future to test URNLookupController on a more granular basis. The tests we have provide good coverage but they're more high-level than I'd like.